### PR TITLE
#29: Changing MaintenanceEventApi to MaintenanceApi. All mention of "…

### PR DESCRIPTION
**Pull request by MS-ResilienX**
…events" in API content is changed to refer to "maintenance appointments" instead. The semantics are kept the same, just a change to nomenclature, however, since the endpoints were named after "events", this impacts them too.

### CODE REVIEW

**Review by AS-resilienx — COMMENTED** [review]
Will changing the method names affect the Maintenance Service build?

File: MaintenanceApi.yaml [hunk 1]
```diff
@@ -338,59 +338,59 @@ components:
...
           type: string
           format: date-time
-          description: Start time of maintenance event via ISO 8601 date-time format
+          description: Start time of maintenance appointment via ISO 8601 date-time format
           example: "2021-01-30T08:30:00Z"
         endTime:
           type: string
           format: date-time
-          description: End time of maintenance event via ISO 8601 date-time format
+          description: End time of maintenance maintenance appointment via ISO 8601 date-time format
```
[the remark this answers is not in the document]
Comment (MS-ResilienX) [reply MaintenanceApi.yaml]: Fixed.

**Comment by MS-ResilienX** [comment]
> Will changing the method names affect the Maintenance Service build?

Yes. It also affects TMI Display. I have the updates completed in FRAIHM-1567 branch.

**Review by AS-resilienx — APPROVED** [review]
The api changes look good.

**Review by JF-ResilienX — APPROVED** [review]

File: MaintenanceApiChangelist.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Version 1.1.1
+Adding additional statistics relating to repairs. 
+
+Version 1.1.0
+Adding a Maintenance Component endpoint for statistics and state information. 
+
+Version 1.0.0
```
[the remark this answers is not in the document]
Comment (MS-ResilienX) [reply MaintenanceApiChangelist.txt]: Consolidated to 0.1.0 per slack conversation.

**Review by GD-ResilienX — APPROVED** [review]

**Pull request merged**